lib/graphql/schema.ts

```diff
@@ -855,6 +855,7 @@ export type Transaction = {
   e2eId?: Maybe<Scalars['String']>,
   mandateNumber?: Maybe<Scalars['String']>,
   fees: Array<TransactionFee>,
+  splits: Array<TransactionSplit>,
```

Comment:
I'm not sure if those other comments are necessary in this file, but maybe we should leave a comment as to what `splits` is?

Reply:
Necessary or not, they are auto generated from `description` fields in our backend, which adds them as documentation programmatically not only to our schema when you inspect it (for example with GraphQL Playground), but also on our live documentation: https://kontist.dev/docs/#schema-reference 🙂We could argue that some of the existing descriptions are self-explanatory, but we need to keep in mind that external developers, or new kontist developers could still be interested in information that seem redundant to us. 

From a glance it's not clear to me what these "splits" are even if I know the existing backend structure and that they look a lot like existing "transaction meta", so I'd say a bit of description could be useful 🙂 

For instance in the backend taxYearSettingsResolvers, we added a couple of descriptions like "Individual tax-related settings per year" which are now included in the schema, and as a result deployed here in the docs: https://kontist.dev/docs/#taxyearsetting

Reply:
I agree it's a good practice to add some descriptions 👍 However, I'm struggling a bit with making a clear but still short description for the field. I mean, we can't squeeze and put the entire feature with all its details in one field description 🤷‍♂so what do you think guys, smth like this `A list of 2 elements containing metadata for split parts of the transaction` would be meaningful enough? or should I be more verbose/specific? @hihuz @francisauwerda

Reply:
Cool @hihuz , that's really good to know, thanks for the explanation! I agree, let's add some descriptions to our BE.

I came up with this:
`Information regarding the two separate transactions created when splitting the parent transaction`
But now I'm thinking your suggestion is good too. 😬 Just put anything and we can always improve it if it still looks confusing.

Reply:
I like yours too @francisauwerda let's let @hihuz pick from the options as an independent viewer (not in the context of the feature) lol

Reply:
Hah, so I have to choose 😛 I think there's no need to be too specific about it, what is the reason we are splitting these metadata for? it's to separate by year or something like that right?

Reply:
you split a transaction into 2 parts - giving each its own amount, category and userSelectedBookingDate - which would lead to more accurate money buckets calculations

Reply:
ok, I guess "Metadata resulting from splitting a transaction" makes sense to me 🤔 not sure if proper english 😄 Otherwise let's go with Francis' suggestion 🙂

Reply:
ok, so let @francisauwerda decide now 😆 kidding, I compiled both your options to `Metadata of separate pseudo-transactions created when splitting the parent transaction`